### PR TITLE
add project brief description

### PR DESCRIPTION
**Pull request by core-man**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- 修正教程中的错误或不清晰的描述
+- 修正reST文件语法错误导致的显示问题
+- 调整章节结构，使得教程条理更清晰
+- 建议、编写或者整理其他地震学常用软件的教程
+
+可以通过如下几种方式参与教程的维护：
+
+1. 通过 GitHub 修改教程源码并提交 Pull Request
+2. 在 GitHub 上提交 Issue
+3. 发送相关建议或投稿至 `seisman.info@gmail.com` | `coreman.seism@gmail.com`
```
Comment (seisman) [inline CONTRIBUTING.md:19]: Add this one https://github.com/seismo-learn/software/discussions?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## 分支模型
+
+项目中存在如下长期分支：
+
+- `main`: 主分支，对应最新版本的教程
+- `gh-pages`: 用户存放网页的分支，自动更新，无需人工修改
```
Comment (seisman) [inline CONTRIBUTING.md:26]:
```suggestion
- `gh-pages`: 存放网页的分支，自动更新，无需人工修改
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (core-man) [reply CONTRIBUTING.md:19]: Shall we remove emails?

Comment (seisman) [reply CONTRIBUTING.md:19]: I think so.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,12 @@
-# software
+# 地震学常用软件教程
 
 ![build](https://github.com/seismo-learn/software/workflows/Build%20Website/badge.svg)
 
+本文档主要介绍地震学常用软件，包括简介、安装、用法等，既可以作为入门读物，也可以作为日常参考。
```
Comment (seisman) [inline README.md:5]:
```suggestion
本文档主要介绍地震学常用软件，包括简介、安装、用法等。
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+- 修正错别字、语句不通等
+- 修正教程中的错误或不清晰的描述
+- 修正reST文件语法错误导致的显示问题
+- 调整章节结构，使得教程条理更清晰
+- 建议、编写或者整理其他地震学常用软件的教程
+
+可以通过如下几种方式参与教程的维护：
+
+1. 通过 GitHub 修改教程源码并提交 Pull Request
+2. 在 GitHub 上提交 Issue
```
Comment (seisman) [inline CONTRIBUTING.md:18]:
```suggestion
2. 在 GitHub 上[提交 Issue](https://github.com/seismo-learn/software/issues)
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+3. 在 GitHub Discussions 留言: General | Ideas | Q&A
```
Comment (seisman) [inline CONTRIBUTING.md:19]:
```suggestion
3. 在 [GitHub Discussions](https://github.com/seismo-learn/software/discussions) 留言
```

**Review by seisman — APPROVED** [review]
Looks good.

BTW, you may want to read these tips about commit messages https://chris.beams.io/posts/git-commit/, and one tip is 

image

**Comment by core-man** [comment]
> Looks good.
> 
> BTW, you may want to read these tips about commit messages https://chris.beams.io/posts/git-commit/, and one tip is
> 
> image

Okay. I will read it when I finish my tutorial next Monday.

**Comment by core-man** [comment]
#7  addressed~

**Pull request merged**